apps/meteor/app/apps/server/bridges/livechat.ts

```
@@ -135,6 +135,28 @@ export class AppLivechatBridge extends LivechatBridge {
 		return true;
 	}
 
+	protected async findRoomsByAgentId(agentId: string, appId: string): Promise<ILivechatRoom[]> {
+		this.orch.debugLog(`The App ${appId} is looking for livechat rooms associated with agent ${agentId}`);
+
+		if (!agentId) {
```

Wouldn't be better to throw an error if the agent id was not provided?